### ISSUE TITLE
"dbld/rules tarball" fails due to bison

### ISSUE THREAD

**Issue by czanik** [syslog-ng/syslog-ng#3461]
When trying to run ```dbld/rules tarball``` it fails due to bison:
```
configure: WARNING: bison is found, but your bison version 3.0.4 is not recent enough, at least 3.4.2 is required
configure: error: syslog-ng requires bison 3.4.2 or later. Your source tree seems to be from git, which doesn't have the bison generated files (like cfg-grammar.c). Please install/upgrade bison or use a distribution tarball.
make: *** [dbld/rules:101: tarball-ubuntu-bionic] Error 1
```
I even tried to re-build the container image, but it did not help. I guess the problem is related to: https://github.com/syslog-ng/syslog-ng/commit/4b9d6f332964c20ad99ddeacb4c30014fcbe042a

**Comment by gaborznagy** [comment]
Yes, since #2526 is merged, the minimum required bison version is `3.4.2`.
Our tarball generating image in dbld is bases on ubuntu:bionic, so it's an old version as you see.
@bazsi opened a PR #3455, but unfortunately, the package building is broken on every platform except debian sid.

**Comment by czanik** [comment]
Well, even that seems to be broken, as ```dbld/rules tarball-debian-sid``` fails with:

```
cd "../../syslog-ng-3.29.1.175.g60d6f7f/lib/jsonc" && doxygen
/bin/bash: doxygen: command not found
```

I even tried fedora-32, but the same problem.

**Comment by gaborznagy** [comment]
AFAIK tarball generation in dbld were not supported on every platform, that's why there is a separate TARBALL_IMAGE.
The above issue comes from this fact, we simply didn't install tarball-related packages (e.g. doxygen) on every platform.

This also means, even after #3455 is merged, targets like `tarball-ubuntu-bionic` and `tarball-debian-sid` won't work.
I'll ask about the use case for `dbld rules tarbal-<image>`, because this command is contrary to aforementioned one-tarball-image principle.

**Comment by bazsi** [comment]
I had the impression that #2526 contained all changes required to build the tarball too. That doesn't seem to be the case as the default tests don't try to build the tarball. 

Let me try to create a minimal PR that resolves this regression.

**Comment by bazsi** [comment]
This should be resolved once #3455 is merged.
